Dockerfile

```diff
@@ -27,8 +27,9 @@ RUN apt-key add /tmp/nodesource.gpg.key && apt-get update -qq \
 RUN apt-get install -y fonts-liberation libappindicator3-1 libasound2 libatk-bridge2.0-0 \
                        libatspi2.0-0 libgtk-3-0 libnspr4 libnss3 libx11-xcb1 libxss1 \
                        libxtst6 xdg-utils phantomjs lsb-release
-RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
-    dpkg -i google-chrome-stable_current_amd64.deb && \
+RUN wget https://github.com/webnicer/chrome-downloads/raw/master/x64.deb/google-chrome-stable_75.0.3770.142-1_amd64.deb \
```

Comment:
Since Google doesn't maintain old chrome versions I had to set this one rather than the official FTP. I tried to keep using the official but upgrading the chromedriver wasn't fixing the issue, so I had to set this instead.

Reply:
Yeah, none of the chromedriver is working with chrome v76. We can't depend on this source for long right. But, for now, we can surely work with this. @jywarren can we merge this?

Reply:
This source is commited to keep the .deb files there. But we could maintain one of our own. Just in case

Reply:
Yeah, maintaining a copy seems like a good idea. But, we will also look forward to using the official source soon, so let's use this one for now.

Reply:
This is great, and lets us switch to different versions under our own decision making process instead of being tied to Google's release schedule, esp since chromedriver stability per version doesn't seem ironclad. Thanks!